### DISCUSSION TITLE
Latest v3 firmware breaks wyze-bridge

### DISCUSSION THREAD

**Discussion by jeremyenos** [mrlt8/docker-wyze-bridge#234] [Q&A] [answered]
Not certain which version specifically did it, but updating firmware today ended my wyze-bridge functionality for the v3.
I updated to latest wyze-bridge container from today as well.  Here is the error I see in the log:

🏁 STARTING ALL (2) CAMERAS
2021/11/15 22:55:27 [Stable Cam] IOTC_ER_DEVICE_OFFLINE
2021/11/15 22:55:27 [Stable Cam] 💀 Camera is offline. Will retry again in 10s.
2021/11/15 22:55:27 [Chick cam] IOTC_ER_DEVICE_OFFLINE
2021/11/15 22:55:27 [Chick cam] 💀 Camera is offline. Will retry again in 10s.

**Comment by mrlt8** [comment, marked as answer]
Does setting the fresh_data flag help? 
See #217

**Reply by jeremyenos** [reply]
Worked great.  Note that I didn't test recreating the container w/o the fresh_data=1 set, so a lot of variables changed at once- but either way, this appears to have worked.  Thank you!!
Is there a way to do that w/o recreating the container instance?
